Xamarin.Forms.Core/Element_StyleSheets.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
 using System.IO;
+using System.Linq;
```

Comment:
I see a couple of LINQ queries happening here. Can we do without? It might have impact on the performance here.

Reply:
I removed the .concat, but left the .Any(), hope that's ok